ui/src/components/Tasks/GenericTaskForm.js

```diff
@@ -43,6 +43,17 @@ class GenericTaskForm extends React.Component {
     this.props.handleSubmit(this.addToQueue.bind(this, true))();
   }
 
+  clearCurrentJSFormParameters() {
+    localStorage.removeItem(`current${this.props.taskData.type}Parameters`);
```

Comment:
You could make a getter for the local storage key instead — it's the logic that is worth abstracting IMO.

```js
get storageKey() {
  return `current${this.props.taskData.type}Parameters`;
}
```

Reply:
Oh thats nice :)